broader-topics/ecotax.md

```diff
@@ -0,0 +1,65 @@
+# SPECIFICATIONS - ECOTAX
+
+## WHAT IS THE ECOTAX
+
+The ecotax is a contribution in France to finance the collect, the recycling and the depollution of a piece of furniture or used electrical and electronic equipment. 
+The ecotax is not a tax per se, but a fixed mandatory contribution (the actual term is "eco-participation") that must be paid by the customer.
+Then it’s donated by the seller to a collection agency. 
+
+The amount of the ecotax is fixed and does not depend on the product price.
+The ecotax is subject to VAT.
+
+## WHERE IS THE ECOTAX IN PRESTASHOP
+
+The ecotax can be toggled in the "Tax Options" section from the _International > Taxes_ page.
+Once enabled, a new drop-down is displayed containing all the tax rules, allowing the merchant to choose the VAT rate that will be applied on the ecotax.
+But this drop-down should rather be displayed on the product page as the ecotax VAT rate should be same as the VAT product rate (see improvement [#22827](https://github.com/PrestaShop/PrestaShop/issues/22827)).
+If the ecotax is enabled, then a field is displayed in the Back Office's "add / edit product" page, so that the merchant can input the amount of included ecotax in the product's price.
```

Comment:
Maybe precise that the field of ecotax is "tax included" ?

Reply:
Yes, good idea, I added it here: https://github.com/PrestaShop/prestashop-specs/pull/188/commits/943c2aaee6ca250a5f5269b9832fba2324f58c7e